src/lib.rs

```diff
@@ -142,4 +145,13 @@ impl Copperline {
         self.history.clear()
     }
 
+
+    /// Clears the screen
+    pub fn clear_screen(&mut self) -> Result<usize, Error> {
+        match self.term.acquire_raw_mode() {
```

Comment:
This looks like it can be solved with `and_then`, instead of having the `Err(e) => Err(e)` case match around.